drivers/hmis/app/graphql/types/date.rb

```diff
@@ -0,0 +1,19 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class Date < BaseScalar
+    description 'A Date object, transporated as a string'
+
+    def self.coerce_input(input_value, _context)
+      Date.parse(input_value)
```

Comment:
Do we need to catch empty input values?
```suggestion
     return nil unless input_value.present?

      Date.parse(input_value)
```

Reply:
`input_value` should never be null, since this shouldn't be called unless a non-null value was passed to the endpoint; But it's probably wise to add something like that just in case